build-tools/packages/build-cli/src/commands/merge/branches.ts

```diff
@@ -55,6 +48,11 @@ export default class MergeBranch extends BaseCommand<typeof MergeBranch> {
 			required: true,
 			multiple: true,
 		}),
+		creds: Flags.string({
+			description: "Username and email id for GitHub auth",
+			required: true,
+			multiple: true,
```

Comment:
This should be false, no?

Reply:
Oh I see... I suggest separate flags. That's more obvious, and otherwise order matters, which is confusing. You can parse the two flags into a single object in the run method and then pass that around as creds. E.g.

```ts
interface GitHubCredentials {
  username: string, 
  email: string,
}

...

const creds: GitHubCredentials = { username: flags.username, email: flags.email };

```